src/types.ts

```diff
@@ -1777,15 +1777,98 @@ export type TranslationLanguages =
   | 'zh'
   | 'zh-TW';
 
+export type EndpointNames =
```

Comment:
maybe `EndpointName` would be better (singular <> plural)

Reply:
good point, fixed